### DISCUSSION TITLE
Determine core usage of multithreading

### DISCUSSION THREAD

**Discussion by dcorson-ticino-com** [jamulussoftware/jamulus#1783] [Ideas]
Is it possible to determine which cores are used for the multi-threading feature on a server?

I have an ARM processor with 2 low-power cores and four high-performance cores.
The older multi-threading code used only the high-power cores, the new rc2 code is using all the cores, but apparently the low-power cores are being overwhelmed and the high-power cores underwhelmed. i.e. at 15 users the low-power cores are running at 55% at max core speed and the high-power cores at under 10% at idle core speed.

I do not know how the system will react when the low-power cores saturate and would like to be able to use only the cores 2-5 which are the high-power cores.
It is strange that I didn't notice this with rc1 (but I am no longer sure I really tested this)

**Comment by hoffie** [comment]
Jamulus-wise, I don't think there are such options. `htop` or something should probably display it though? You may also be able to limit Jamulus to use the high-power cores only (`taskset`). I think this may even work thread-specific.

**Reply by dcorson-ticino-com** [reply]
That works !  Great.
~~Only problem, I am seeing with rv2 about 2x the core usage than I had the last time I tried, I think that was beta1.
All with delay panning on.~~
EDIT: after a reset all works well.

**Reply by softins** [reply]
Interesting. The multi-threading implementation changed between 3.8.0beta2 and 3.8.0rc1. The new one certainly seems better on Intel. It would be interesting to see the comparison on Arm.
